File: server/udp.py
#!/usr/bin/env python3
import sys
import socket

if len(sys.argv) < 3:
	print("Wrong invocation of udp.py", file=sys.stderr)
	sys.exit(1)

address = sys.argv[1]
port    = int(sys.argv[2])
payload = sys.stdin.buffer.read()
readsize= 1024

sock = socket.socket(socket.AF_INET, socket.SOCK_DGRAM);
sock.bind((address,0))
sock.settimeout(0.1)
sock.sendto(payload, (address, port))

try:
	(data,addr) = sock.recvfrom(1024)
	sys.stdout.buffer.write(data)
except:
	print("Connection failed", file=sys.stderr)

File: server/udp.php
#!/usr/bin/env php
<?php

if ( count($argv) < 0 )
{
	fprintf (STDERR, "Wrong invocation of udp.php");
	exit(1);
}

$address = $argv[1];
$port    = $argv[2];
$payload = fgets(STDIN);
$readsize= 1024;
$read_address = empty($argv[3]) ? $address : $argv[3];
$read_port = $port;

$socket = socket_create(AF_INET, SOCK_DGRAM, SOL_UDP);
socket_bind($socket, $address);
socket_getsockname($socket,$read_address,$read_port);
socket_set_timeout($socket,0,1000);
socket_sendto($socket, $payload, strlen($payload), 0, $address, $port);


@socket_recvfrom($socket, $data, $readsize, MSG_DONTWAIT, $read_address, $read_port);
print($data);